gapcloser/1.12-r6/Dockerfile

```diff
@@ -0,0 +1,40 @@
+#################################################################
+# Dockerfile
+#
+# Version:          1
+# Software:         gapcloser
+# Software Version: 1.12-r6
+# Description:      a novel short-read assembly method that can build a de novo draft assembly for the human-sized genomes
+# Website:          http://soap.genomics.org.cn/gapcloser.html
+# Tags:             Genomics
+# Provides:         gapcloser 1.12-r6
+# Base Image:       biodckr/biodocker
+# Build Cmd:        docker build --rm -t biodckrdev/gapcloser v1.12-r6/.
+# Pull Cmd:         docker pull biodckrdev/gapcloser
+# Run Cmd:          docker run --rm -it biodckrdev/gapcloser <options> <files>
+#################################################################
+
+# Set the base image to Ubuntu
+FROM biodckr/biodocker
```

Comment:
same remarks, should use biocontainers base image and use LABEL for metadata (+ license info)